ui-tests/src/test/resources/features/integrations/integration-import-export.feature

```diff
@@ -95,13 +85,15 @@ Feature: Integration - Import Export
     And sleep for jenkins delay or "3" seconds
     # start integration and wait for published state
     And start integration "Integration_import_export_test"
+    Then check visibility of "Integration_import_export_test" integration details
+
     And navigate to the "Integrations" page
+    And Integration "Integration_import_export_test" is present in integrations list
+
+    And sleep for "300000" ms
     Then wait until integration "Integration_import_export_test" gets into "Running" state
 
-    # validate salesforce contacts
-    Then check that contact from SF with last name: "Jackson" has description "Red Hat"
-    # clean-up in salesforce
-    Then delete contact from SF with last name: "Jackson"
+    Then check that last slack message equals "Red Hat" on channel "test"
```

Comment:
Isn't last message already "Red Hat" from previously run integration? maybe you should send something else to be sure that the message is new?